doc/classes/RID.xml

```diff
@@ -4,7 +4,7 @@
 		Handle for a [Resource]'s unique ID.
 	</brief_description>
 	<description>
-		The RID type is used to access the unique integer ID of a resource. They are opaque, which means they do not grant access to the associated resource by themselves. They are used by and with the low-level Server classes such as [RenderingServer].
+		The RID type is used to access the unique integer ID of a [Resource]. They are opaque, which means they do not grant access to the associated resource by themselves. They are used by and with the low-level server classes, such as [DisplayServer] and [RenderingServer].
```

Comment:
> integer ID of a [Resource]

It's not always true, server can have RIDs of internal objects which are not `Resource`s.

Reply:
Well, ain't that some misleading information sitting there for plenty of time...

Reply:
Took this occasion to change the above descriptions a bit more. See what you think.